### ISSUE TITLE
`LoadAsync(FileInfo)` and `LoadAsync(string)` build a `Stream` and leave it opened

### ISSUE THREAD

**Issue by icalvo** [EPPlusSoftware/EPPlus#604]
An example of why this is problematic would be:

```csharp
using var package = new ExcelPackage()) {
  await package.LoadAsync("file.xlsx");
  // ...
}

File.Delete("file.xlsx"); // This throws an Exception because the file is still opened
```

In this is scenario, async would needed (so `new ExcelPackage("file.xlsx")` is not a a possible workaround).

**Workaround:** build the stream outside and use `LoadAsync(Stream)`.

**Proposals:**
- Get rid of these overloads
- Return the stream
- Return an `IDisposable` that disposes the stream when disposed
- Keep track of all these streams and dispose them when the `ExcelPackage` instance is disposed

**Comment by BryceBarbara** [comment]
Is this safe to close now?

**Comment by JanKallman** [comment]
This should now be fixed in version 5.8.7
